presto-main/src/main/java/com/facebook/presto/security/AccessControlManager.java

```diff
@@ -554,6 +554,18 @@ public void checkCanRevokeRoles(TransactionId transactionId, Identity identity,
         }
     }
 
+    @Override
+    public void checkCanSetRole(TransactionId transactionId, Identity identity, String role, String catalogName)
+    {
+        requireNonNull(identity, "identity is null");
+        requireNonNull(role, "role is null");
+        requireNonNull(catalogName, "catalog is null");
+        CatalogAccessControlEntry entry = getConnectorAccessControl(transactionId, catalogName);
```

Comment:
During the "writing a doc" phase we have decided that access control is completely on connector. If we want to change this assumption - we need to change it everywhere. For how if you don't want some user to be an admin - you can revoke admin role for him in connector.